rc/base/ctags.kak

```diff
@@ -9,9 +9,10 @@ declare-option -docstring "list of paths to tag files to parse when looking up a
 define-command -params ..1 \
     -shell-script-candidates %{
         realpath() { ( path=$(readlink "$1"); cd "$(dirname "$1")"; printf "%s/%s\n" "$(pwd -P)" "$(basename "$1")" ) }
+        search() { [ -f "$1" ] && realpath "$1" || [ "$1" != "${1#/}" ] || ( while [ "$(pwd)" != "/" ]; do cd ..; [ -f "$1" ] && realpath "$1" && break; done ) }
```

Comment:
I don't have any stake in the maintenance of this script, but I find this style of doing lots of things on one line rather hard to read.

Reply:
@occivink I totally agree. When changing someone else's code I always try to follow the prevailing style. If the file should be reformatted I can do it.

Reply:
should one search past `#$HOME`? This may cause permission error on some machines, where users can't access files beyond their home directory.

Reply:
@andreyorst In such an enviroment, what does `cd ..` do when called in `$HOME`?